system/security/shared/seclib.hpp

```diff
@@ -116,6 +116,8 @@ enum SecUserStatus
 };
 
 
+const static int scPasswordExpired = -1;
```

Comment:
Trivial: I assume the sc is for static const.  We don't really use that naming style inside the main platform - although I know esp does.